docs/work/content/04_content/ausblick.tex

```diff
@@ -0,0 +1,55 @@
+\chapter{Ausblick}
+
+	Im Folgenden werden einige zusätzliche Funktionalitäten erläutert, die das Projekt \textit{Travlyn} erweitern. Diese Features waren jedoch nicht im festgelegten Scope für diese Arbeit, könnten jedoch in zukünftigen Projekten umgesetzt werden. 
```

Comment:
Siehe Ende evaluations kapitel--> vllt sowas wie:
Um die zugrunde liegende Vision dieses Projekts zu erfüllen könnten weitere Funktionalitäten implementiert werden, die nicht teil des scopes in dieser Entwicklung waren. Im Folgenden wird auf diese Möglichkeiten eingegangen und damit aufgezeigt, was in zukünfigen Projekten und Erweiterungen der Travlyn Applikation umgesetzt werden könnte.